packages/types/src/span.ts

```diff
@@ -235,7 +235,10 @@ export interface Span extends SpanContext {
 
   /**
    * Sets the finish timestamp on the current span.
+   *
    * @param endTimestamp Takes an endTimestamp if the end should not be the time when you call this function.
+   *
+   * @deprecated Use `end()` instead.
```

Comment:
giga-nit, so it is clear that the the class method should be used: 

```suggestion
   * @deprecated Use `.end()` instead.
```